demo/screens/AppNavigator.js

```diff
@@ -0,0 +1,34 @@
+import { createStackNavigator, createAppContainer } from "react-navigation";
```

Comment:
I could possibly even see an argument that this belongs one dir up, since it's not a screen per se.

Reply:
Yes, I was in doubt about it.